classes/BackupFinder.php

```diff
@@ -87,6 +90,9 @@ private function getBackupDbAvailable(string $backupPath): array
         return $array;
     }
 
+    /**
+     * @return array<int, string>
```

Comment:
Don't change anything, but I think the type 

```
     * @return string[]
```

would have been better here.